schema.graphql

```diff
@@ -433,14 +447,85 @@ type Match {
 }
 
 """
-Docs group: Match
+Docs group: Question
+"""
+type Packet {
+  """
+  A human-readable identifier. Often primarily numeric (e.g., `Packet 1`), but could also be identified by authors (`Chicago` or `Yale + Maryland`), as for packet-submission tournaments.
+  """
+  identifier: String!
+  
+  """
+  The packet number, if that notion is meaningful. May also be contained in the `identifier`, but need not be, e.g. if both `number` and `authors` are specified, the `identifier` will typically only contain one of them (whichever is more important).
+  """
+  number: Int
+  
+  """
+  The authors of the packet. May also be contained in the `identifier`, but need not be, e.g. if both `number` and `authors` are specified, the `identifier` will typically only contain one of them (whichever is more important).
+  """
+  authors: [String]
+
+  """
+  How the packet is expected to be used. An `extra` packet typically consists only of tossups, and is for use in overtime and/or as replacements, as necessary. Note that a packet might be used differently in case of unusual situations.
+  """
+  role: PacketRole
+
+  """
+  Any further details on the packet.
+  """
+  note: String
```

Comment:
`Match` uses `notes` instead of `note`.

Reply:
I think that's really poor usage, since it's a singular field. `notes` should be an array of strings.

That said, we already have it, and I prize consistency, so I made the change (also in `Signature`). If you'd be comfortable changing `Match.notes` to `note` (breaking compatibility, which we're allowed to do in major version bump), I'd probably prefer that.

Reply:
I kind of see "notes" as a collective noun instead of a plural, and I personally prefer it to "note".